test/test_pipeline_baseline.py

```diff
@@ -43,30 +43,51 @@ def get_expected_calls(baseline, binding, stderr=False):
     return commands
 
 
-def check_calls_match(expected_calls, actual_calls):
-    err_str = 'Mismatch between expected and actual subprocess calls.\r\
-        Expected: {}\rActual: {}'
-    assert expected_calls == actual_calls, err_str.format(
-        expected_calls, actual_calls)
+def format_arg(arg, bindings):
+    for (k, v) in bindings.items():
+        arg = arg.replace(v, '{' + k + '}')
+    return arg
 
 
-def make_fake_python_output(output_dir):
-    # Create an empty 'fake_output_api.py' in the output_dir. Do not invoke
-    # 'touch' command with subprocess.call() because it's mocked.
-    final_output_dir = os.path.join(output_dir, 'library-v1-gapic-gen-python')
-    if not os.path.exists(final_output_dir):
-        os.makedirs(final_output_dir)
-    with open(os.path.join(final_output_dir, 'fake_output_api.py'), 'w'):
-        pass
+def format_args(args, bindings):
+    return ' '.join([format_arg(arg, bindings) for arg in args])
+
+
+def format_calls(calls, bindings):
+    return '\n'.join([format_args(args, bindings)
+                      for (name, (args,), kwargs) in calls])
 
 
-def make_fake_java_output(output_dir):
-    # Create an empty 'FakeOutputApi.java' in the output_dir. Do not invoke
+def check_calls_match(expected_calls, actual_calls, bindings):
+    mismatch_text = 'Mismatch between expected and actual subprocess calls.\n'
+
+    def format_err(exp, act):
+        return mismatch_text + 'Expected:\n' + exp + '\nActual:\n' + act
+    exp = format_calls(expected_calls, bindings)
+    act = format_calls(actual_calls, bindings)
+    assert exp == act, format_err(exp, act)
+
+
+_DUMMY_FILE_DICT = {
+    'java': 'MyApi.java',
+    'python': 'my_api.py',
+    'go': 'my_api.go',
+    'ruby': 'my_api.rb',
+    'php': 'MyApi.php',
+    'csharp': 'MyApi.cs',
+    'nodejs': 'my_api.js'
+}
+
+
+def make_fake_gapic_output(output_dir, language):
+    # Create an empty 'README.md' in the output_dir. Do not invoke
```

Comment:
oops some documentation may be out of date... it actually uses a filename from the table right above. I'll fix the comment.